app/src/main/java/com/ece493/group5/adjustableaudio/storage/Encrypter.java

```diff
@@ -0,0 +1,156 @@
+package com.ece493.group5.adjustableaudio.storage;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Base64;
+import android.util.Log;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.KeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.KeyGenerator;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+public class Encrypter {
+
+    private static final int KEY_SIZE = 256;
+    private static final String SHARED_PREFS_FILE = "KEY_STORAGE";
+    private static final String SECRET_KEY_IDENTIFIER = "S_KEY";
+    private static final String IV_IDENTIFIER = "IV";
+
+    private static SecretKey secretKey = null;
+    private static IvParameterSpec iv = null;
```

Comment:
My only concern here is that IV is not a descriptive of the name, and it is the Roman numeral for 4. I had to look up that it mean initialization vector. This could confuse the TA and Lab instructor and lessen the readability of the code.